### PR TITLE
visual updates

### PR DESCRIPTION
**Pull request by thrilldavis**
fixed some images showing on same line as last text and added alt text

### CODE REVIEW

File: authentication-guides/workday/README.md [hunk 1]
```diff
@@ -107,41 +107,41 @@ Run the **`Activate Pending Security Policy Changes`** task to activate permissi
 
 Universal search for **`Register API Client for Integrations`**
 
-![https://developer.moveworks.com/static/3f4721c823875fbc16322fb8cc12f78c/410f3/Untitled_10.png](https://developer.moveworks.com/static/3f4721c823875fbc16322fb8cc12f78c/410f3/Untitled_10.png)
+![search for register api client for integrations](https://developer.moveworks.com/static/3f4721c823875fbc16322fb8cc12f78c/410f3/Untitled_10.png)
```
Comment (ajaymerchia) [inline authentication-guides/workday/README.md:110]: This was moved to the help docs site.

Comment (ajaymerchia) [reply authentication-guides/workday/README.md:110]: https://help.moveworks.com/docs/workday-access-requirements

File: authentication-guides/workday/README.md [hunk 2]
```diff
@@ -107,41 +107,41 @@ Run the **`Activate Pending Security Policy Changes`** task to activate permissi
...
 
 From the **`View API Clients`** view, click on the **`API Clients for Integrations`** tab. Pick out the API Client you just created
 
-![https://developer.moveworks.com/static/703d5272d024109eb4b30d808292e8a6/c6bbc/inspect_client.png](images/inspect_client.png)
+![view api clients with red box around "api clients for integrations"](images/inspect_client.png)
 
 From the related actions menu, select **`Manage Refresh Tokens for Integrations`**
 
-![https://developer.moveworks.com/static/6a2406d2a729e02c9e1f214ee8a6ea35/c6bbc/Untitled_14.png](https://developer.moveworks.com/static/6a2406d2a729e02c9e1f214ee8a6ea35/c6bbc/Untitled_14.png)
+![manage refresh tokens page](https://developer.moveworks.com/static/6a2406d2a729e02c9e1f214ee8a6ea35/c6bbc/Untitled_14.png)
```
Comment (ajaymerchia) [inline authentication-guides/workday/README.md:132]: Moved to https://help.moveworks.com/docs/workday-access-requirements

File: authentication-guides/workday/README.md [hunk 3]
```diff
@@ -107,41 +107,41 @@ Run the **`Activate Pending Security Policy Changes`** task to activate permissi
...
 
 Add the ISU Account you created earlier to the API Client
 
-![https://developer.moveworks.com/static/b1bab6dffdf67d31c35640c840c0f17b/c6bbc/isu_to_client.png](https://developer.moveworks.com/static/b1bab6dffdf67d31c35640c840c0f17b/c6bbc/isu_to_client.png)
+![showing adding the isu account created earlier](https://developer.moveworks.com/static/b1bab6dffdf67d31c35640c840c0f17b/c6bbc/isu_to_client.png)
```
Comment (ajaymerchia) [inline authentication-guides/workday/README.md:136]: Moved to https://help.moveworks.com/docs/workday-access-requirements

File: authentication-guides/workday/README.md [hunk 4]
```diff
@@ -107,41 +107,41 @@ Run the **`Activate Pending Security Policy Changes`** task to activate permissi
...
 
 Select **`Generate Refresh Token`**
 
-![https://developer.moveworks.com/static/2a3fbc8565c2adb8678d2388338278d6/c6bbc/gen_refresh_token.png](https://developer.moveworks.com/static/2a3fbc8565c2adb8678d2388338278d6/c6bbc/gen_refresh_token.png)
+![showing generate refresh token buttom](https://developer.moveworks.com/static/2a3fbc8565c2adb8678d2388338278d6/c6bbc/gen_refresh_token.png)
```
Comment (ajaymerchia) [inline authentication-guides/workday/README.md:140]: Moved to https://help.moveworks.com/docs/workday-access-requirements

File: authentication-guides/workday/README.md [hunk 5]
```diff
@@ -107,41 +107,41 @@ Run the **`Activate Pending Security Policy Changes`** task to activate permissi
...
 
 Note your  new refresh token.
 
-![https://developer.moveworks.com/static/48972e313711f799a7f8172586f256ca/c6bbc/refresh_token.png](https://developer.moveworks.com/static/48972e313711f799a7f8172586f256ca/c6bbc/refresh_token.png)
+![new refresh token page, copy this for your own notes](https://developer.moveworks.com/static/48972e313711f799a7f8172586f256ca/c6bbc/refresh_token.png)
```
Comment (ajaymerchia) [inline authentication-guides/workday/README.md:144]: Moved to https://help.moveworks.com/docs/workday-access-requirements

**Review by ajaymerchia — DISMISSED** [review]

File: authentication-guides/workday/README.md (hunk 4 above)
Comment (thrilldavis) [reply authentication-guides/workday/README.md]: brought image over locally

File: authentication-guides/workday/README.md (hunk 5 above)
Comment (thrilldavis) [reply authentication-guides/workday/README.md]: brought image locally

File: authentication-guides/workday/README.md (hunk 3 above)
Comment (thrilldavis) [reply authentication-guides/workday/README.md]: brought image locally

File: authentication-guides/workday/README.md (hunk 2 above)
Comment (thrilldavis) [reply authentication-guides/workday/README.md]: brought image locally

File: authentication-guides/workday/README.md (hunk 1 above)
Comment (thrilldavis) [reply authentication-guides/workday/README.md]: brought image locally

**Review by sarthaksrinivas — APPROVED** [review]
looks good to publish

**Review by sarthaksrinivas — APPROVED** [review]

**Pull request merged**